app/localization/translated/be.json

```diff
@@ -292,16 +325,31 @@
   "RegistrationPage.tokenNotProvided": "запрашэнне не было прадастаўлена ў параметрах URL",
   "RegistrationPage.visit": "Наведайце",
   "RegistrationPage.welcome": "Дабро запрашаем,",
+  "RemoveIndexModal.cancelButtonText": "Адмяніць",
+  "RemoveIndexModal.contentHeaderMessage": "Вы ўпэўнены, што жадаеце выдаліць усю інфармацыю з ElasticSearch?",
+  "RemoveIndexModal.headerRemoveIndexModal": "Выдаліць індэкс",
+  "RemoveIndexModal.removeButtonText": "Выдаліць",
+  "RemoveIndexModal.removeErrorNotification": "Something went wrong...",
```

Comment:
Translate

Reply:
I didn't translate it because task description haven't default message for this notification, so it is a temporary message